docs/configure-artifact-repository.md

```diff
@@ -623,7 +623,7 @@ configuring the default artifact repository described previously.
 ## Artifact Streaming
 
 With artifact streaming, artifacts don’t need to be saved to disk first. Artifact streaming is only supported in the following
-artifact drivers: S3 (v3.4+), Azure Blob (v3.4+), HTTP (v3.5+), and Artifactory (v3.5+).
+artifact drivers: S3 (v3.4+), Azure Blob (v3.4+), HTTP (v3.5+), Artifactory (v3.5+), and OSS (v3.6+).
```

Comment:
we should probably change this to a table given there's 5+ drivers here now

Reply:
Or add "Streaming" to the top level table on this page. Which might be a bit dated too as well

Reply:
This whole page probably wants some effort putting in. This was a minimum update just for correctness.

Reply:
Oh for sure, I've been gradually reviewing large pieces of it in PRs (#13376 and https://github.com/argoproj/argo-workflows/pull/12467#pullrequestreview-2273917077).

> This was a minimum update just for correctness.

I did approve the PR as well, i.e. making this comment optional